build-tools/extract_resources.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Path of stacks directory in the registry
+STACKS_DIR=/build/stacks
```

Comment:
Need this to override `STACKS_DIR`:
```suggestion
STACKS_DIR=${STACKS_DIR:-/build/stacks}
```